src/main/apiClient.ts

```diff
@@ -620,6 +617,48 @@ export default class ApiClient {
     }
   }
 
+  signupWithWebAuthn(params: SignupWithWebAuthnParams, auth?: AuthOptions) {
+    if (navigator.credentials && navigator.credentials.get) {
```

Comment:
According to [this page](https://developers.google.com/web/updates/2018/03/webauthn-credential-management), feature detection should be done this way instead:
```suggestion
    if (window.PublicKeyCredential) {
```

Reply:
That's interesting! Thanks